rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -828,15 +828,30 @@ namespace rsx
 	{
 		// Get timestamp, and convert it from microseconds to nanoseconds
 		const u64 t = get_guest_system_time() * 1000;
-		if (t != timestamp_ctrl)
+
+		if (t <= vtimestamp)
```

Comment:
This will murder zcull when we have a long queue (some games can queue almost a thousand entries at once). I suggest moving this to another PR for now as we think of better solutions.